Lib/_collections_abc.py

```diff
@@ -85,6 +85,9 @@ def _f(): pass
 dict_items = type({}.items())
 ## misc ##
 mappingproxy = type(type.__dict__)
+def get_framelocalsproxy():
```

Comment:
```suggestion
def _get_framelocalsproxy():
```

Reply:
Hmm, this is failing the test, maybe forgot to fix the typo?